i18n/english.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable max-len */
+
+const scanner = {
+  disable_scarf: "This dependency could collect data against your will so think to disable it with the env var: SCARF_ANALYTICS",
```

Comment:
```suggestion
  disable_scarf: "This dependency could collect data against your consent so think to disable it with the env var: SCARF_ANALYTICS",
```
Something like this ? The original statement sound weird to me